charts/datadog/templates/_containers-common-env.yaml

```diff
@@ -87,6 +88,16 @@
 {{- end }}
 {{- end }}
 {{- end }}
+
+{{- if .Values.datadog.otlp.logs }}
```

Comment:
I think the {{ if ...}} is not needed since {{ with ...}} will be executed only if the value is not empty or I suppose false. 
See: https://helm.sh/docs/chart_template_guide/control_structures/

Otherwise we can only use a "if" since the value will always be "true"

Reply:
yeah that seems right:
> Note that we removed the if conditional from the previous exercise because it is now unnecessary - the block after with only executes if the value of PIPELINE is not empty.

I have removed the additional `if`